plugins/grace-hopper/tests/tests.rs

```diff
@@ -65,16 +66,16 @@ fn full_plugin_with_multiple_hwmon_sensors() {
     let file_path_average = root.path().join("hwmon3/device/power1_average");
     let file_path_interval = root.path().join("hwmon3/device/power1_average_interval");
     std::fs::create_dir_all(file_path_info.parent().unwrap()).unwrap();
-    std::fs::write(file_path_info, "CPU Power Socket 2").unwrap();
-    std::fs::write(file_path_average, "64000000").unwrap();
+    std::fs::write(file_path_info, "CPU Power Socket 0").unwrap();
```

Comment:
Can you use several sockets in the test, to check that it works fine? (add new devices for, let's say, socket 2).